### DISCUSSION TITLE
Glyph support

### DISCUSSION THREAD

**Discussion by ghost** [MissingCore/Music#70] [Ideas]
I'm not sure what it would do exactly but its only natural for Glyph support.

**Comment by cyanChill** [comment]
Glyph support would probably just be a "rave" with lights. Nothing OS has a `Music Visualization` (they spelt it as visuali`s`ation) option that you can toggle.

Currently, it doesn't work with this app because we don't play audio through the music notification (that's how the visualization works I assume; even though it claims it works for audio played through the speakers, it really means audio played though the music notification).

After the `v1.0.0` release, I plan on refactoring the code so it does go through the notification (we didn't do this initially because I didn't want to figure it out at the time how to dynamically update the list of tracks through the notification, since we're using a package that abstracts that logic and that its "simpler" in the way we currently do it).
